### PR TITLE
fix for 9.0 uat bugs

### PR DESCRIPTION
**Pull request by SidelineCory24**
Dual Merge
- [FE](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2306)

Addresses:
- AO/CDO being unable to export External CDA ABL View

### CODE REVIEW

File: talentmap_api/bureau/views/available_bidders.py [hunk 1]
```diff
@@ -1,5 +1,4 @@
 import logging
-import coreapi
```
Comment (scott062) [inline talentmap_api/bureau/views/available_bidders.py:2]: Did something prompt this?

Comment (SidelineCory24) [reply talentmap_api/bureau/views/available_bidders.py:2]: it's my auto save linter thingy, adding back

Comment (SidelineCory24) [reply talentmap_api/bureau/views/available_bidders.py:2]: it may not be used at all which is why my linter removes it🤷‍♂️

Comment (scott062) [reply talentmap_api/bureau/views/available_bidders.py:2]: Nah its okay if its not being used - just curious if it was linter error or something

**Review by scott062 — APPROVED** [review]

File: talentmap_api/bureau/views/available_bidders.py (hunk 1 above)
Comment (SidelineCory24) [reply talentmap_api/bureau/views/available_bidders.py]: I put it back, by linter just started doing it out of nowhere when i do cmd + s, but not on when I let it autosave on it's on

File: talentmap_api/bureau/views/available_bidders.py [hunk 2]
```diff
@@ -32,7 +32,7 @@ def get(self, request):
 
 
 class AvailableBiddersCSVView(APIView):
-    permission_classes = [Or(isDjangoGroupMember('post_user'), isDjangoGroupMember('bureau_user')), ]
+    permission_classes = [Or(isDjangoGroupMember('post_user'), isDjangoGroupMember('bureau_user'), isDjangoGroupMember('cdo'), isDjangoGroupMember('ao_user')), ]
```
Comment (elizabeth-jimenez) [inline talentmap_api/bureau/views/available_bidders.py:35]: 🦄

**Pull request merged**